package.json

```diff
@@ -159,6 +159,16 @@
         "command": "quarkusTools.deployToOpenShift.short",
         "title": "Deploy to OpenShift",
         "icon": "icons/openshift.png"
+      },
+      {
+        "command": "qute.validation.enabled.toggle.on",
+        "title": "Qute Validation Enabled",
+        "icon": "$(star-full)"
+      },
+      {
+        "command": "qute.validation.enabled.toggle.off",
+        "title": "Qute Validation Disabled",
+        "icon": "$(star-empty)"
```

Comment:
Is there another icon that has 2 states and better represents validation ? I was thinking of maybe search/search-stop but that could too easily be confused as some kind of search :\